components/helpers.py

```diff
@@ -85,20 +85,24 @@ async def _make_request(self, method: str, url: str, data: dict = None) -> dict:
         async with aiohttp.ClientSession(headers=self.headers) as session:
             response = None
             response_content = None
+            response_status = None
+            content_type = None
             if method in ["POST", "PATCH"]:
                 async with session.request(method=method, url=url, json=data) as r:
                     response_content = await r.content.read()
+                    response_status = r.status
+                    content_type = r.headers.get('content-type', '')
             else:
                 async with session.request(method=method, url=url, params=data) as r:
                     response_content = await r.content.read()
+                    response_status = r.status
+                    content_type = r.headers.get('content-type', '')
```

Comment:
Change "r" to "response" for readability purposes. It will allow future readers what the response variable is at a quick glance.
```suggestion
            if method in ["POST", "PATCH"]:
                async with session.request(method=method, url=url, json=data) as response:
                    response_content = await response.content.read()
                    response_status = response.status
                    content_type = response.headers.get('content-type', '')
            else:
                async with session.request(method=method, url=url, params=data) as response:
                    response_content = await response.content.read()
                    response_status = response.status
                    content_type = response.headers.get('content-type', '')
```